src/pydvl/value/loo/naive.py

```diff
@@ -1,26 +1,32 @@
-from typing import OrderedDict
-
 import numpy as np
 
-from pydvl.reporting.scores import sort_values
-from pydvl.utils import Utility, maybe_progress
+from pydvl.utils import SortOrder, Utility, maybe_progress
+from pydvl.value import ValuationResult, ValuationStatus
```

Comment:
I think it would be safer here to import directly from `pydvl.value.results` to avoid any possible future circular import errors.

Reply:
```suggestion
from pydvl.value.results import ValuationResult, ValuationStatus
```